kubernetes/cray-dhcp-kea/files/dhcp-helper.py

```diff
@@ -146,27 +146,48 @@ def get_ipxe_boot_filename(ipxe_settings_file):
 
     ipxe_filename['ipxe'] = ipxe_settings.get('cray_ipxe_binary_name', '')
     ipxe_filename['ipxe_debug'] = ipxe_settings.get('cray_ipxe_debug_binary_name', '')
+    ipxe_filename['ipxe_aarch64'] = ipxe_settings.get('cray_ipxe_aarch64_binary_name', '')
+    ipxe_filename['ipxe_aarch64_debug'] = ipxe_settings.get('cray_ipxe_aarch64_debug_binary_name', '')
```

Comment:
nit: since the new vars have the arch in the name, should the old vars get refactored to also have their arch or does that creep into other code repos?